### ISSUE TITLE
Missing documentation on CreateFeedSpecification.feedOptions

### ISSUE THREAD

**Issue by johnkw** [amzn/selling-partner-api-docs#1953]
According to the `CreateFeedSpecification` documentation at https://github.com/amzn/selling-partner-api-docs/blob/main/references/feeds-api/feeds_2021-06-30.md#feedoptions there are:
`Additional options to control the feed. These vary by feed type.`

But what are the options?

As least one option seems to exist, at least internally, which is that the `POST_FLAT_FILE_LISTINGS_DATA` feed can return either a CSV file or an Excel file.  But this is not documented.  Seller Support actually claims it's not in the SP-API at all.  Either way, it should be documented whether these options exist, or don't currently exist but may be added in the future.

**Comment by jenilew** [comment]
Hi JohnKW, 

Feed options are detailed in the Feed Type Values topic (where we describe each individual type). See https://developer-docs.amazon.com/sp-api/docs/feed-type-values for more information. 

We will only document feed options that currently exist where there is not an XSD or Schema available to reference. 

Hope this helps!

**Comment by johnkw** [comment]
Thanks, so, that does sort of help. I had not actually seen that page (or forgot about it), but it is also available here: https://github.com/amzn/selling-partner-api-docs/blob/main/references/feeds-api/feedtype-values.md

That page needs to be linked to from https://github.com/amzn/selling-partner-api-docs/blob/main/references/feeds-api/feeds_2021-06-30.md#createfeedspecification where it currently says "The feed type." it should also say "See feedtype-values.md for the list of feed types."

However, this bug 1953 was actually just about feedOptions missing documentation.  Strangely enough though I do see one feedType on there that mentions a feedOptions.  It's the very last one and is India-only apparently.

It would be good to have far more documentation on those feedTypes.  For every one that lacks a feedOptions, it should be documented that there are "No feedOptions for this feedType."

Also not a documentation issue, but if you specify an invalid feedOptions, the API should reject it with a friendly error message telling you what feedOptions are allowed.

**Comment by gzfcgsh** [comment]
> so can you tell me what to fill in the param feedOptions , I Can't find any examples， only kown it is a Map<String, String>， but what is the key and value？

me too!

**Comment by lashaLobjanidzeXentral** [comment]
### Options for `UPLOAD_VAT_INVOICE`
There are required `FeedOptions` for this FeedType, see next error message:

`Please provide metadata:shippingid or metadata:orderid,metadata:invoicenumber,metadata:totalamount,metadata:totalvatamount`

The FeedOptions for MWS for [SubmitFeed](https://docs.developer.amazonservices.com/en_US/feeds/Feeds_SubmitFeed.html) looks like this:

'FeedOptions=metadata:orderid=111-2222222-3333333;metadata:TotalAmount=39.99;metadata:TotalVATAmount=2.62;metadata:InvoiceNumber=2022-IN0000000'

### The question is now, how does SP-API expect the parameter to be passed? 
Documentation only provides `Map<String, String>`, I was not able to find any example. I assume it should look like this:

`Array(
    "metadata:orderid=111-222-333",
    "metadata:TotalAmount=99.99",
    "metadata:TotalVATAmount=1.62",
    "metadata:InvoiceNumber=INV0000000",
)`

**Comment by johnkw** [comment]
This is still broken. https://developer-docs.amazon.com/sp-api/docs/feeds-api-v2021-06-30-reference#createfeedspecification needs to link over to the list of feed types, and every feed type needs to document the precise list of exactly what feedoptions are allowed, including specifically saying none are allowed if that's the case.

**Comment by johnkw** [comment]
Please turn this horrible bot off and only close the github tickets as the actual issues are fixed. @adrian-amz @amazon-auto @chapmanjw @amvreddy-amazon @jenilew @parvathm

**Comment by johnkw** [comment]
Please turn this horrible bot off and only close the github tickets as the actual issues are fixed. @adrian-amz @amazon-auto @chapmanjw @amvreddy-amazon @jenilew @parvathm

**Comment by jason-quick** [comment]
I agree with @johnkw, it has been almost a year and a half since he pointed out this substantial dead end in the documentation and it really should have been addressed by now.

**Comment by KhorneHoly** [comment]
When will this finally be fixed?
